x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/wizard.tsx

```diff
@@ -62,7 +60,7 @@ const DefinePivotStep: SFC<DefinePivotStepProps> = ({
       <div ref={definePivotRef} />
       {isCurrentStep && (
         <Fragment>
-          <DefinePivotForm onChange={setPivot} overrides={{ ...pivotState, jobConfig }} />
```

Comment:
@elastic/ml-ui TS is now complaining because `jobConfig` is not allowed in the type for this prop, and best I can tell it's not used inside `<DefinePivotForm>` anywhere, so I've removed it. Seem right to y'all?

Reply:
LGTM! Looks like a leftover from a refactor!